src/qservkafka/storage/votable.py

```diff
@@ -474,8 +817,16 @@ async def store(
             raise UploadWebError.from_exception(e) from e
         finally:
             await generator.aclose()
-        return EncodedSize(
-            rows=encoder.total_rows,
-            data_bytes=encoder.encoded_size,
-            total_bytes=encoder.total_size,
-        )
+
+        if isinstance(encoder, VOParquetEncoder):
+            return EncodedSize(
+                rows=encoder.total_rows,
+                data_bytes=encoder.encoded_size,
+                total_bytes=encoder.encoded_size,
+            )
+        else:  # VOTable
+            return EncodedSize(
+                rows=encoder.total_rows,
+                data_bytes=encoder.encoded_size,
+                total_bytes=encoder.total_size,
```

Comment:
If you add a `total_size` property to `VOParquetEncoder` as a synonym for `encoded_size`, you could unify this part of the function too.

Reply:
Yea that looks much better thanks!